backend/twitch-api/resource/stream-tags.js

```diff
@@ -14,108 +12,55 @@ const { snakeKeys } = require('js-convert-case');
  * @property {string} description - A dictionary that contains the localized descriptions of the tag
  */
 
-function mapTwitchTag(tag) {
+const mapTwitchTag = (tag) => {
     return {
-        id: tag.tag_id,
-        isAuto: tag.is_auto,
-        name: tag.localization_names["en-us"],
-        description: tag.localization_descriptions["en-us"]
+        id: tag.id,
+        isAuto: tag.isAuto,
+        name: tag.getName("en-us"),
+        description: tag.getDescription("en-us")
     };
-}
-
-async function getAllStreamTags(cursor) {
-    const client = twitchApi.getClient();
+};
 
+const getAllStreamTags = async () => {
     try {
-        let response = {};
-
-        if (cursor == null) {
-            response = await client.callApi({
-                type: TwitchAPICallType.Helix,
-                url: "tags/streams"
-            });
-        } else {
-            response = await client.callApi({
-                type: TwitchAPICallType.Helix,
-                url: "tags/streams",
-                query: {
-                    after: cursor
-                }
-            });
-        }
+        const client = twitchApi.getClient();
+        const response = await client.helix.tags.getAllStreamTagsPaginated().getAll();
 
-        if (response == null || response.data == null || response.data.length < 1) {
-            return null;
+        if (response) {
+            /**@type {TwitchStreamTag[]} */
+            return response.filter(tag => !tag.isAuto).map(tag => mapTwitchTag(tag));
         }
-
-        return response;
     } catch (error) {
         logger.error("Failed to get all stream tags", error);
         return null;
```

Comment:
I know this is a previously exiting function. Just wanted to mention the returning an empty array instead of null thing again. Not sure how it would affect what is calling this though.

Reply:
Fixed.